RoboFile.php

```diff
@@ -393,7 +439,7 @@ public function qaCodeSniffer(array $filesToCheck, $opts = ['severity' => 'all']
       'tests/_output',
       'tests/_support/_generated',
       'vendor',
-      'vendor-prefixed',
+      'vendor-prefixed/gregwar/captcha/demo',
```

Comment:
Looks like this change is incorrect, we don't need to sniff the whole vendor-prefixed folder, it should still be ignored.